setup.cfg

```diff
@@ -115,12 +116,16 @@ all =
 # optional (i.e. opt-in) packages, see https://github.com/napari/napari/pull/3867#discussion_r864354854
 optional =
     triangle
+performance =
```

Comment:
I don't think we should have `triangle` in both `optional` and `performance` and it's not clear to me that we want to add yet another extra - we already have so many :< I think we should just put `numba` under `optional`.